AIDevGallery/Pages/HomePage.xaml.cs

```diff
@@ -46,4 +67,31 @@ private async void HandleDiagnosticsSetting(bool isEnabled)
         App.AppData.IsDiagnosticDataEnabled = isEnabled;
         await App.AppData.SaveAsync();
     }
+
+    private string GetFirstSentenceFromDescription(string? description)
+    {
+        if (description == null)
+        {
+            return string.Empty;
+        }
+
+        int i;
+        for (i = 0; i < description.Length; i++)
+        {
+            if (description[i] == '.')
+            {
+                break;
+            }
+        }
```

Comment:
Isn't this just string.IndexOf('.')?

Reply:
Oh, this was existing code I didn't refactor and just copied over. Updated